eng/scripts/Language-Settings.ps1

```diff
@@ -367,7 +367,7 @@ function Get-java-DocsMsDevLanguageSpecificPackageInfo($packageInfo, $packageSou
     if ($packageInfo.DevVersion) {
       $version = $packageInfo.DevVersion
     }
-    $namespaces = Fetch-Namespaces-From-Javadoc $packageInfo.Name $packageInfo.Group $version
+    $namespaces = Fetch-Namespaces-From-Javadoc $packageInfo.ArtifactName $packageInfo.Group $version
```

Comment:
I am 99.99% certain you're already aware of `js` `ArtifactName` being "remove special characters" value:

```
C:/repo/azure-sdk-tools [further-package-props-tests]|>cat ..\azure-sdk-for-js\PackageInfo\azure-identity.json
{
  "Name": "@azure/identity",
  "Version": "4.7.1",
  "DevVersion": null,
  "DirectoryPath": "sdk/identity/identity",
  "ServiceDirectory": "identity",
  "ReadMePath": "sdk/identity/identity/README.md",
  "ChangeLogPath": "sdk/identity/identity/CHANGELOG.md",
  "Group": null,
  "SdkType": "client",
  "IsNewSdk": true,
  "ArtifactName": "azure-identity", <-- this guy
  ```

Reply:
Aha but this is in a specific Java-only function nm.